sigh/lib/sigh/runner.rb

```diff
@@ -216,7 +216,12 @@ def certificate_to_use
     # Downloads and stores the provisioning profile
     def download_profile(profile)
       UI.important "Downloading provisioning profile..."
-      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}.mobileprovision" # default name
+      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
```

Comment:
@hjanuschka, thank you for the review. We need '_tvos' just before .mobileprovision. So, I am not sure if we can append it to [line 219](https://github.com/fastlane/fastlane/pull/8123/files#diff-e9e7336e24f66a051f7c7e07128efd8fR219). 

Would something like this work for you? Or, did you have something else in mind.

```diff ruby
profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
if Sigh.config[:platform].to_s == 'tvos'
    profile_name += "_tvos"
end
- profile_name += '.mobileprovision' unless profile_name.include? 'mobileprovision
+ profile_name += '.mobileprovision'
```